### ISSUE TITLE
Invalid urls for RELEASES file generated for windows updates

### ISSUE THREAD

**Issue by Supamiu** [zeit/hazel#85]
When checking the REALESES file download, with updated links to be able to download nupkg files properly, I saw that the nupkg links were not all edited, and were not mathcing the updates.

**Observed behavior**:

 - My releases: https://github.com/Supamiu/ffxiv-teamcraft/releases
 - The file you get from `https://hazel.supamiu.now.sh/update/win32/7.0.0-beta.7/RELEASES`: 

```
E091039C90E653AC0DCD0D74BB7C50E7DBCC8B42 https://github.com/Supamiu/ffxiv-teamcraft/releases/download/v7.0.0-beta.8/ffxiv-teamcraft-7.0.0-beta7-full.nupkg 79186641
EB4A4C0ED8595A7984EEF41E782BE87328517585 ffxiv-teamcraft-7.0.0-beta8-delta.nupkg 284017
B1FED2E7FB4CC4C94DE6A9622EFE261ECE350B45 ffxiv-teamcraft-7.0.0-beta8-full.nupkg 79014815
```

**Expected behavior**:

I expected the urls to all be replaced with the corresponding github links, so the windows updater can properly download them from github.

**Comment by nathanvogel** [comment]
I had the same problem and made a fix. Feel free to use [my branch](https://github.com/nathanvogel/hazel/tree/bug/fix86) or copy-paste my changes of #87, if it doesn't get merged quickly enough.

**Comment by nathanvogel** [comment]
(this can now be closed as it's fixed by #87 )
